lib/prompt.js

```diff
@@ -5,14 +5,25 @@ const { promisify } = require('util');
 const read = promisify(require('read'));
 
 async function prompt(message, options) {
+    let value;
+
     // Read input
-    let value = await read({
-        prompt: message,
-        silent: options.silent,
-        replace: options.replace,
-        input: options.input,
-        output: options.output,
-    });
+    // Manage timeout
+    try {
+        value = await read({
+            prompt: message,
+            silent: options.silent,
+            replace: options.replace,
+            input: options.input,
+            output: options.output,
+            timeout: options.timeout,
+        });
+    } catch (err) {
+        if (err.message !== 'timed out' || !options.default) {
+            throw err;
+        }
+        value = options.default;
```

Comment:
I wouldn’t set the default value on timeout. The default value should only be used if the user type enter without typing anything.

Reply:
I think this match some use case and give more freedom on using the lib, I do not think there is hard argument in both case, as they are both defendable, I guess.

So my preference is current implementation but I can change it if this is not acceptable for you.

Reply:
I would error out, that’s how read work. It might be dangerous to assume the default value if the user is not focused on the prompt and is doing other things.

Reply:
What about adding a timeoutToDefaullt boolean option to let the user choose?

Reply:
I implemented it with timeoutToDefault, what do you think?

Reply:
Seems alright, but I would name the option to useDefaultOnTimeout.